mirai-core-mock/src/contact/MockGroup.kt

```diff
@@ -39,6 +41,15 @@ public interface MockGroup : Group, MockContact, MockMsgSyncSupport {
     override val owner: MockNormalMember
     override val botAsMember: MockNormalMember
     override val announcements: MockAnnouncements
+    public val honorMembers: MutableMap<GroupHonorType, MockNormalMember>
+
+    /**
+     * 更改拥有群荣耀的群成员
```

Comment:
```suggestion
     * 更改拥有群荣耀的群成员.
```

KDoc 在 render 之后会删除换行, 类似 Markdown 的解析, 写 KDoc 请务必写完后预览 rendered 效果

Reply:
是在哪生成KDoc

Reply:
右键注释块，选 Render

Reply:
ok 原来是这个